src/rest-server/src/models/v2/job/k8s.js

```diff
@@ -107,6 +116,11 @@ const convertState = (state, exitCode, retryDelaySec) => {
 };
 
 const convertFrameworkSummary = (framework) => {
+  // check fields which may be compressed
+  if (framework.status.attemptStatus.taskRoleStatuses == null) {
```

Comment:
Seems no need taskRoleStatuses for FrameworkSummary and job list? So you do not need to decompress it?